testing/libc++/APKBUILD

```diff
@@ -0,0 +1,94 @@
+# Contributor: Dmitry Golovin <dima@golovin.in>
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Dmitry Golovin <dima@golovin.in>
+pkgname=libc++
+pkgver=4.0.0
+pkgrel=0
+_llvmver=${pkgver%%.*}
+pkgdesc="A new implementation of the C++ standard library, targeting C++11"
+url="http://libcxx.llvm.org/"
+arch="all"
+license="UOI-NCSA"
+depends_dev="${pkgname}abi-dev=$pkgver-r$pkgrel"
+makedepends="cmake llvm-dev>=$_llvmver llvm-static>=$_llvmver clang>=$_llvmver"
+checkdepends="lit"
+subpackages="$pkgname-dev ${pkgname}abi:abi ${pkgname}abi-dev:abi_dev"
+source="http://releases.llvm.org/$pkgver/libcxx-$pkgver.src.tar.xz
+	http://releases.llvm.org/$pkgver/libcxxabi-$pkgver.src.tar.xz
+	http://releases.llvm.org/$pkgver/libunwind-$pkgver.src.tar.xz
```

Comment:
Does it really need libunwind sources? We already have separate package for llvm-libunwind.